### ISSUE TITLE
Questions assignment 4

### ISSUE THREAD

**Issue by rozniaka** [Aelphy/ISC#12]
I have 2 questions for the 4th assignment: 

1. In the Intro to Numpy part there is stated: "For every problem provide examples of their usage." Does it pertain to the all the tasks from 1 to 14? 

2. Mikhail, could you clarify what is meant by "Train your perceptron and report the results." in exercise Perceptron? Are we supposed to implement the multi-label classifier from scratch? Or can we use a ready implementation, such as sklearn.linear_model.Perceptron?

**Comment by Aelphy** [comment]
1. Yes (that basically means that we gonna play a game: if you didn't test your solutions properly and I will find a case where it fails - I won)
2. You are supposed to implement your classifier from scratch.

**Comment by rozniaka** [comment]
Thanks for fast reply! 
1. Is example a use case of this function, e.g. "calculation of mean per R,G,B channels" or something else? 
2. Can we use cycles in Perceptron? Were the forbidden for the whole notebook or only the Numpy part?

**Comment by Aelphy** [comment]
1. No, it is a particular example: `np.random.random((512, 512, 3))`
and testing that your function outputs expected result
2. You can. Only in `numpy` part

**Comment by rozniaka** [comment]
Thanks again. I know now everything about the Numpy part. 
In the perceptron part... are we supposed to predict 2 or 3 classes? Even the perceptron tutorial (link in the week 4 folder) focuses only on binary classification

**Comment by Aelphy** [comment]
That's because two classes case is easy to visualize.
You need to separate all 3 classes.
